packages/angular_devkit/architect/BUILD.bazel

```diff
@@ -66,10 +68,9 @@ ts_project(
     data = JSON_FILES,
     module_name = "@angular-devkit/architect",
     deps = [
+        ":node_modules/@angular-devkit/core",
         "//:node_modules/@types/node",
         "//:node_modules/rxjs",
-        "//packages/angular_devkit/core:core_rjs",
```

Comment:
This nicely prevents the cross-relative imports (like we decided in MUC!). It also fixes/improves some `.d.ts` output generation where TS currently emits e.g. `import('../core')`